crypto/s2n_certificate.c

```diff
@@ -595,7 +595,7 @@ int s2n_get_cert_from_cert_chain(const struct s2n_cert_chain_and_key *chain_and_
     return S2N_SUCCESS;
 }
 
-int s2n_get_cert_der(const struct s2n_cert *cert, const uint8_t **out_cert_der, uint32_t *cert_length)
+int s2n_cert_get_cert_der(const struct s2n_cert *cert, const uint8_t **out_cert_der, uint32_t *cert_length)
 {
```

Comment:
For all your "cert_get_cert" -- maybe drop the second "cert"? So for example, "s2n_cert_get_der" instead of "s2n_cert_get_cert_der"

Reply:
For this API I can change to `s2n_cert_get_der`, the rest I think needs to remain as is as it looses its meaning:
```
s2n_cert_get_cert_from_cert_chain
s2n_cert_get_cert_chain_length
```